kimchi/src/circuits/constraints.rs

```diff
@@ -688,15 +688,13 @@ impl<F: PrimeField + SquareRootField> Builder<F> {
             // First we sum over the lookup table size
             let mut lookup_domain_size: usize = lookup_tables
                 .iter()
-                .map(
-                    |LookupTable { data, id: _ }| {
-                        if data.is_empty() {
-                            0
-                        } else {
-                            data[0].len()
-                        }
-                    },
-                )
+                .map(|lt| {
+                    if lt.data().is_empty() {
+                        0
+                    } else {
+                        lt.data()[0].len()
```

Comment:
This is the width. There is a method for this.

Reply:
Addressed.